index.html

```diff
@@ -0,0 +1,97 @@
+</head>
+
+<body>
+    <header>
+        <nav class="upper-nav">
+            <ul class="upper-ul">
+                <li><a href="#"><img src="./images/facebook.png" alt="facebookIcon"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="https://www.google.com/">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <img class="logo" src="./images/logo.png" alt="logo">
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="about.html" class="nav-list" id="about-link">About</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main-container">
+        <h1 class="main-heading">Hello! sharing world</h1>
+        <h2 class="main-sec-heading">CREATIVE CITY SUMMIT 2015</h2>
+        <p class="main-par">Join us in Waterloo, 2015 for an exciting itinerary of speakers, peer
+            presentations, facilitated discussions, networking, and local culture offerings from the City of Waterloo
+            and regional partners. In-person and virtual tickets are on sale now!</p>
+        <p class="bold">
+            15 November 2015 FROM 08:00am
+        </p>
+        <p class="bottom-text">@ Canada , the City of Waterloo</p>
+    </main>
+
+    <section class="main-program" id="program">
+        <div class="wrapper">
+            <div class="program-container">
+                <div class="program-title flex-container">
+                    <h3 class="main-section-h1">Main Program</h3>
+                    <div class="orange-underline"></div>
+                </div>
+                <div class="program-categories">
+                </div>
+                <a class="btn-mobile-program">SEE THE WHOLE PROGRAM</a>
+                <button type="button" class="btn-program">Join CC Summit 2015</button>
+            </div>
+        </div>
+    </section>
+
+
+    <!--SPEAKER-->
+    <section class="speaker">
+        <h2 class="main-section-h1 ">Featured Speakers</h2>
+        <div class="orange-underline"></div>
+        <div class="speaker-container" id="container">
+        </div>
+        <button type="button" class="more-btn">More <img src="./images/arrow_down.png" alt="arrow_down"></button>
+
+    </section>
+    <!-- sponsers -->
+    <section class="partner hidden">
```

Comment:
- [OPTIONAL] Kindly consider adding this background-color `#414246` to this section, and then `gray` color to the logos to make it similar to the behance design.